CHANGELOG.md

```diff
@@ -111,6 +111,64 @@ relevant information. ProtoQuill uses them in order to pass Ast information as w
 the query is Static or Dynamic into execute and prepare methods. In the future, Scala2-Quill may be enhanced
 to use them as well.
 
+# 3.10.1 
+
+#### Migration Notes:
+
+Datastax standard configuration file format and properties (HOCON) are used to provide cassandra driver configuration
+
+Sample HOCON:
+```hocon
+MyCassandraDb {
+       preparedStatementCacheSize=1000
+       keyspace=quill_test
+
+        session {
+            basic.contact-points = [ ${?CASSANDRA_CONTACT_POINT_0}, ${?CASSANDRA_CONTACT_POINT_1} ]
+            basic.load-balancing-policy.local-datacenter = ${?CASSANDRA_DC}
+            basic.request.consistency = LOCAL_QUORUM
+            basic.request.page-size = 3
+        }
+
+}
+```
+
+The `session` entry values and keys are described in the datastax documentation:
+[Reference configuration](https://docs.datastax.com/en/developer/java-driver/4.13/manual/core/configuration/reference/) 
+
+
+The ZioCassandraSession constructors:
+
+```scala
+ val zioSessionLayer: ZLayer[Any, Throwable, Has[CassandraZioSession]] =
+  CassandraZioSession.fromPrefix("MyCassandraDb")
+run(query[Person])
+  .provideCustomLayer(zioSessionLayer)
+```
+
+Additional parameters can be added programmatically:
+```scala
+ val zioSessionLayer: ZLayer[Any, Throwable, Has[CassandraZioSession]] =
+  CassandraZioSession.fromContextConfigCassandraContextConfig(LoadConfig("MyCassandraDb").withValue("keyspace", ConfigValueFactory.fromAnyRef("data")))
```

Comment:
Should this be just `CassandraZioSession.fromContextConfig` ?

Reply:
fixed